crm_stage_probability/views/crm_lead.xml

```diff
@@ -11,7 +11,7 @@
             >
                 <field name="is_stage_probability" invisible="1" />
                 <button
-                    class="btn btn-link"
+                    class="ps-0 ps-md2 btn btn-link"
```

Comment:
I think this needs to be "ps-md-2" instead of "ps-md2"
![image](https://github.com/OCA/crm/assets/1799080/1765346a-57ef-463a-9f9e-d610b5c33803)
![image](https://github.com/OCA/crm/assets/1799080/00bc564f-5ce6-41ce-a74c-98199a7eab52)

Reply:
Thank you for your review @CRogos I made the changes requested.
Best Regards